proto/scripts/protoc_plugin.go

```diff
@@ -34,11 +34,11 @@ func main() {
 	vanity.ForEachFile(files, vanity.TurnOffGoUnrecognizedAll)
 	vanity.ForEachFile(files, vanity.TurnOffGoUnkeyedAll)
 	vanity.ForEachFile(files, vanity.TurnOffGoSizecacheAll)
-	vanity.ForEachFile(files, vanity.TurnOffGoEnumStringerAll)
+	vanity.ForEachFile(files, vanity.TurnOnMarshalerAll)
+	vanity.ForEachFile(files, vanity.TurnOnUnmarshalerAll)
 	vanity.ForEachEnumInFiles(files, HandleCustomEnumExtensions)
 	vanity.ForEachFile(files, SetPackagePrefix("github.com/syncthing/syncthing"))
-
-	vanity.ForEachMessageInFiles(files, HandleCustomExtensions)
+	vanity.ForEachFile(files, HandleFile)
```

Comment:
What's the intention of this change? On the surface it looks like it doesn't actually change anything, just calls `ForEachMessageInFiles` on single files instead of all of them at once. And it makes the diff of `HandleCustomExtensions` almost impossible to read due to the indentation change.

Reply:
So we now pass the file ref to HandleCustomExtensions and in the handling of (device_id) =  true do not prepend the type with package name if the file we are processing is in the same package. This is to prevent lib/protocol from importing itself.

Reply:
Also, if you prepend `?w=1` to the url (or set it via the settings cog) you can ignore whitespace changes which makes indent changes a breeze to look at.

Reply:
Wow, that looks much nicer now (and makes the changes very clear) - thanks for the tip :)